### ISSUE TITLE
[Question] Accessibility Compliance Status - WCAG 2.1 AA

### ISSUE THREAD

**Issue by joshcampbell191** [surveyjs/survey-library#7240]
### Are you requesting a feature, reporting a bug or asking a question?
Asking a question

### What is the current behavior?
Can you please confirm the accessibility compliance status SurveyJS in conformance to WCAG 2.1 AA? I noticed that accessibility tests are targeting WCAG 2.1 AA but failed to find anything detailing the results.

**Comment by andrewtelnov** [comment]
@joshcampbell191 Yes, it is our target. If there is a scenario where we fail to support  WCAG 2.1 AA then we will consider it as a bug and we will work on the fix.

Thank you,
Andrew

**Comment by ElenaGorbatkova** [comment]
Hi @joshcampbell191,

Thank you for your question. I'd like to provide further details on our current accessibility maintenance practices.

As you can see in the [Accessibility Statement](https://surveyjs.io/accessibility-statement), we conduct a series of tests using Axe and Testcafe. These tests cover 15 question types and keyboard navigation functionality, ensuring their compliance with several standards represented in the code as a list of [axeTags](https://www.deque.com/axe/core-documentation/api-documentation/#axe-core-tags), which include ["wcag2a", "wcag2aa", "wcag21a", "wcag21aa", "best-practice", "section508", "wcag412"].

Each WCAG version and level has a set of rules. Our tests check all the [rules](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md) corresponding to a specific version and level. The '[runOnly](https://github.com/surveyjs/survey-library/blob/d61f4f2b6682e35bad974066d5cfe11479d69fd5/accessibilityTests/helper.ts#L86C1-L86C1)' property restricts the rules executed, based on names or tags.

As we run tests on every PR submitted, we are committed to ensuring that the question types and navigation functionality specified in the Accessibility Statement comply with all rules falling under WCAG 2.0 and WCAG 2.1 (Level A and AA). If you encounter an accessibility-related issue while interacting with any of the tested question types, please do let us know!  

For more information on how these tests are set up, please refer to the [axe API guides](https://www.deque.com/axe/core-documentation/api-documentation/#overview).

I hope that helps!

Elena
